components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesBookmarksStorageTest.kt

```diff
@@ -205,9 +205,7 @@ class PlacesBookmarksStorageTest {
         try {
             bookmarks.importFromFennec(path)
             fail("Expected v0 database to be unsupported")
-        } catch (e: PlacesException) {
-            // This is a little brittle, but the places library doesn't have a proper error type for this.
-            assertEquals("Unexpected error: Can not import from database version 0", e.message)
+        } catch (e: PlacesApiException) {
```

Comment:
The above note says that this is brittle because there isn't an error type specifically for import errors.  We could add a variant for that, but I'm not so sure about that.

It seems to me that calling `importFromFennic()` and catching any `PlacesApiException` subclass is a fine way to handle errors here.  Am I missing something?

Reply:
No, I agree that's fine - this code is dead anyway IIUC (I don't think Fenix still does that import)